ebu_tt_live/bindings/validation/timing.py

```diff
@@ -176,7 +183,10 @@ def _semantic_postprocess_timing(self, dataset, element_content):
             # This requires calculation based on the timings in its children.
 
             # All timing containers are complexTypes so we can call orderedContent safely
-            children = filter(lambda item: isinstance(item, TimingValidationMixin), [x.value for x in self.orderedContent()])
+            # but we don't want to bother with explicitly badly timed elements so filter
+            # them out.
+            children = filter(lambda item: isinstance(item, TimingValidationMixin) \
```

Comment:
Not a fan with how long this line is.

Reply:
OK, this was done though it is not showing as "outdated" on the GitHub pull request page strangely - @tomrosier if you're happy with it please could you "approve" in a review?